<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="CategoryLabel">
        <property name="text">
         <string>Category:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="CategoryComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="EntryLabel">
        <property name="text">
         <string>Entry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EntryLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAddToCategories">
        <property name="text">
         <string>Add To Categories</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
